rdf_differ/utils.py

```diff
@@ -0,0 +1,67 @@
+"""
+utils.py
+Date: 09/07/2020
+Author: Mihai Coșleț
+Email: coslet.mihai@gmail.com
+"""
+import os
+from pathlib import Path
+from typing import Union
+
+from rdf_differ import defaults
+
+
+def get_envs() -> dict:
```

Comment:
As a suggestion, you can move this function to a new module, like `config.py`. It's going to be more clear what this function does without even checking the code :)